### PR TITLE
optimize bulk index creation

### PR DESCRIPTION
**Pull request by mfussenegger**
was always set to true (except in tests)

### CODE REVIEW

**Comment by seut** [comment]
ack

**Comment by dobe** [comment]
this breaks cross version compat

File: sql/src/main/java/org/elasticsearch/action/admin/indices/create/BulkCreateIndicesRequest.java [hunk 1]
```diff
@@ -126,7 +111,7 @@ public void readFrom(StreamInput in) throws IOException {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
-        out.writeBoolean(ignoreExisting);
+        out.writeBoolean(true);  // used to be ignoreExisting setting; here for binary compat. - remove for next feature release
```
Comment (seut) [inline sql/src/main/java/org/elasticsearch/action/admin/indices/create/BulkCreateIndicesRequest.java]: pls make this comment a TODO, so we can spot it for the next feature release

**Comment by seut** [comment]
ack

**Pull request merged**